tools/packaging/packaging.psm1

```diff
@@ -22,11 +22,6 @@ function Start-PSPackage {
         [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg")]
         [string[]]$Type,
 
-        # Generate windows downlevel package
```

Comment:
How do you make it mandatory?  What happens when you are producing a `deb`, `pkg`, etc?